hazelcast/src/main/java/com/hazelcast/monitor/NodeState.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.monitor;
+
+import com.hazelcast.cluster.ClusterState;
+import com.hazelcast.internal.management.JsonSerializable;
+import com.hazelcast.version.Version;
+
+public interface NodeState extends JsonSerializable {
```

Comment:
That's a good observation. I just followed the convention they is use there - all of the classes are divided into interface and impl there:
https://github.com/hazelcast/hazelcast/tree/master/hazelcast/src/main/java/com/hazelcast/monitor